packages/shared/src/components/squads/SquadsDirectoryHeader.tsx

```diff
@@ -27,9 +27,8 @@ export const SquadsDirectoryHeader = ({
     >
       <div className="relative z-1 flex flex-col items-center">
         <SourceBetaIcon
-          width="auto"
```

Comment:
On SVGs, `width="auto"` is not allowed. However, adding a `w-auto` class is rendering a different result. With the `w-full`, it goes back to normal.